src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -346,7 +353,9 @@ public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
             }
 
             string json = JsonConvert.SerializeObject(result);
-            if (json.Length > ScriptConstants.MaxTriggersStringLength)
+
+            // TODO Avoiding max request size limit operartion for Kubernetes Environment
```

Comment:
Recommend removing this TODO, since likely we'll never want the limit to apply in Kubernetes cases, since there is no FE limit to deal with?

Reply:
Done!